### DISCUSSION TITLE
don't get the --stream option

### DISCUSSION THREAD

**Discussion by maddes8cht** [LostRuins/koboldcpp#109] [Q&A]
In the Oobabooga webinterface i can see the Ai's Answer "streaming in" while it is being generated. In Koboldcpp, even on the console i just see a counter of the generated tokens, then - boom - the answer is showed up as a whole.
Which then makes sense to reduce the Answers length to 80 Tokens, but i would like to have it the way it is in Oobabooga.

I was hoping the `--stream` option would do this, but it doesn't - what is it meant for, if not this?
Is there a way to get the kind of `stream` i was looking for?

**Comment by LostRuins** [comment]
It does but you need to use the streaming url for it to work.

Try this: `--stream --launch`
